test/test_future_and_futures.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+import imp
+from django.test import TestCase
+
+
+class FutureAndFuturesTestCase(TestCase):
+
+    def test_import_concurrent_py3(self):
+        import concurrent
+        if sys.version_info[0] == 3:
+            # Python 3 is supposed to import its builtin package `concurrent`
+            # instead of the one inside kolibri/dist
+            self.assertTrue('kolibri/dist' not in concurrent.__file__)
+
+    def test_import_future_py2(self):
+        from future.standard_library import TOP_LEVEL_MODULES
+        from kolibri import dist as kolibri_dist
+
+        dist_dir = os.path.realpath(os.path.dirname(kolibri_dist.__file__))
+        if sys.version_info[0] == 2:
+            for module_name in TOP_LEVEL_MODULES:
+                if 'test' in module_name:
```

Comment:
Maybe we can simplify this entire test just by checking if certain modules are in the right place, e.g. if `html.__file__` is not in `kolibri/dist`

Reply:
Do you mean instead of all the standard libraries from `future`, we just pick `html` as an example to run the test?

Reply:
Sorry, forgot to clarify -- all modules defined in `TOP_LEVEL_MODULES`, or whatever we want to ensure is in `dist`

Reply:
In the updated test, I check that if the modules are in the `kolibri/dist` (they are not supposed to be). However, I didn't check if the modules are inside `kolibri/dist/py2only`, because somehow when we ran the tests, all of these modules got installed into `.tox/py2.7/lib/python2.7/site-packages` folder. That is to say, Python finds the modules (i.e. `html`) in system packages instead of `kolibri/dist/py2only` in this specific case. Since when running in other environments, usually we don't have `html` installed in the system, so the path of `html` is not determined. The only thing we can determine is that it shouldn't be inside `kolibri/dist`. This is why I only check if modules are under `kolibri/dist`. Do you think if this test is okay? @aronasorman Thank you!

Reply:
@lyw07 that's because Tox uses a different environment from our built distribution, that's why I mention `usedevelop` :)

Reply:
I see! I tried with removing `usedevelop` but got this error when running `tox -e py2.7`
```
ERROR: invocation failed (exit code 1), logfile: /Users/lingyiwang/Documents/LearningEquality/kolibri/.tox/log/tox-0.log
ERROR: actionid: tox
msg: packaging
cmdargs: ['/Users/lingyiwang/.venvs/kolibri0.7/bin/python', local('/Users/lingyiwang/Documents/LearningEquality/kolibri/setup.py'), 'sdist', '--formats=zip', '--dist-dir', local('/Users/lingyiwang/Documents/LearningEquality/kolibri/.tox/dist')]

Traceback (most recent call last):
  File "setup.py", line 54, in <module>
    sys.argv.remove('--static')
ValueError: list.remove(x): x not in list

ERROR: FAIL could not package project - v = InvocationError('/Users/lingyiwang/.venvs/kolibri0.7/bin/python /Users/lingyiwang/Documents/LearningEquality/kolibri/setup.py sdist --formats=zip --dist-dir /Users/lingyiwang/Documents/LearningEquality/kolibri/.tox/dist (see /Users/lingyiwang/Documents/LearningEquality/kolibri/.tox/log/tox-0.log)', 1)
```

I'm not sure how to fix this, so I used the first suggestion you gave, which is to just add `make staticdeps`

Reply:
The above *could* be fixed, but I think it's okay to keep this simple and then open a new issue to remove `usedevelop` :)

@aronasorman would you accept the current test given that it's already proved that it correctly fails and we just open up an issue that would basically also make the test redundant?